front-end/src/util/requests.js

```diff
@@ -38,6 +38,99 @@ export async function getWorkspaceInfo(workspaceId){
  * 
  * @param {*} workspaceId The workspace's id you'd like to retrieve
  */
-export async function getWorkspaceText(workspaceId){
+export async function getWorkspaceText(workspaceId) {
   return await axios.get(`${process.env.REACT_APP_BACKEND_SERVER}/texts/byWorkspace/${workspaceId}`);
-}
\ No newline at end of file
+}
+
+/**
+ * Add a new workspace to the database for the given user.
+ * 
+ * @param {*} workspaceName The name of the workspace.
+ * @param {*} accountId     The owner of the workspace.
+ * @returns 
+ */
+export async function addNewWorkspace(workspaceName, accountId) {
+  let response;
+  const newWorkspace = {
+    name: workspaceName,
+    owner: accountId,
+    collaborators: [],
+    isPublic: false,
+    texts: [],
+    creationDate: Date.now(),
+    updateDate: null,
+    deleteDate: null
+  };
+
+  await axios.post(`${process.env.REACT_APP_BACKEND_SERVER}/workspaces/add/`, newWorkspace)
+    .then((res) => {
+      response = res;
+    });
+  
+  return response;
+}
+
+/**
+ * Update the given user's list of workspaces.
+ * 
+ * @param {*} id            The id of the user.
+ * @param {*} workspaceIds  The new list of workspaces for the user.
+ */
+export async function updateAccountWorkspaces(id, workspaceIds) {
+  await axios.patch(`${process.env.REACT_APP_BACKEND_SERVER}/accounts/update/${id}`, {workspaces: workspaceIds});
+}
+
+/**
+ * Update the given workspace.
+ * 
+ * @param {*} id        The id of the workspace.
+ * @param {*} workspace This contains any new updates for the given workspace.
```

Comment:
Could you list the expected properties/structure of the workspace object in the comment. Being able to view it there would be very helpful 😊